Assets/MixedRealityToolkit/InputModule/Scripts/Utilities/ControllerFinder.cs

```diff
@@ -42,6 +42,12 @@ public InteractionSourceHandedness Handedness
         protected virtual void OnEnable()
         {
 #if UNITY_WSA && UNITY_2017_2_OR_NEWER
+            if (!MotionControllerVisualizer.ConfirmInitialized())
+            {
+                // The motion controller visualizer singleton could not be found.
+                return;
```

Comment:
remove the controller finder? I don't believe that is needed (I don't see exceptions anywhere else but OnEnable)

Reply:
Yeah, mostly a question about if there will be any perf hit to leaving this unused component in the scene. I don't think there should be much, since it doesn't have an `Update` loop or anything.

Reply:
I don't think there will be any perf issues with leaving it in.